awx/main/isolated/manager.py

```diff
@@ -210,10 +210,22 @@ def check(self, interval=None):
         if status == 'successful':
             status_path = self.path_to('artifacts', self.ident, 'status')
             rc_path = self.path_to('artifacts', self.ident, 'rc')
-            with open(status_path, 'r') as f:
-                status = f.readline()
-            with open(rc_path, 'r') as f:
-                rc = int(f.readline())
+            if os.path.exists(status_path):
```

Comment:
Could this itself raise an error if permission errors, invalid symbolic link, etc. leading up to the path?

Reply:
Oh, this is AWX, blah, I thought this was runner code. This should succeed :+1: